tests/src/test/scala/whisk/core/database/test/ReplicatorTests.scala

```diff
@@ -130,6 +130,17 @@ class ReplicatorTests extends FlatSpec
         }
     }
 
+    // Do cleanups of possibly existing databases
+    val dbs = replicatorClient.dbs().futureValue
+    dbs shouldBe 'right
+    dbs.right.get.filter(dbname => dbname.contains(testDbPrefix)).map(dbName => removeDatabase(dbName))
+
+    val docs = replicatorClient.getAllDocs().futureValue
+    docs shouldBe 'right
+    docs.right.get.fields("rows").convertTo[List[JsObject]]
+        .filter(_.fields("id").convertTo[String].contains(testDbPrefix))
```

Comment:
The problem is, that the test creates databases with the prefix "continous_testDbPrefix", "backup_timestamp_testDbPrefix" and "testDbPrefix".
The testDbPrefix is already a combination of the string "replicatorTests" and the dbPrefix of the environment.
That's why I just use contains instead of handling all cases.